src/components/app/App.js

```diff
@@ -5,257 +5,27 @@ import './App.css';
-            "id": 155328,
-            "node_id": "MDQ6VXNlcjE1NTMyOA==",
-            "avatar_url": "https://avatars1.githubusercontent.com/u/155328?v=4",
-            "gravatar_id": "",
-            "url": "https://api.github.com/users/pelya",
-            "html_url": "https://github.com/pelya",
-            "followers_url": "https://api.github.com/users/pelya/followers",
-            "following_url": "https://api.github.com/users/pelya/following{/other_user}",
-            "gists_url": "https://api.github.com/users/pelya/gists{/gist_id}",
-            "starred_url": "https://api.github.com/users/pelya/starred{/owner}{/repo}",
-            "subscriptions_url": "https://api.github.com/users/pelya/subscriptions",
-            "organizations_url": "https://api.github.com/users/pelya/orgs",
-            "repos_url": "https://api.github.com/users/pelya/repos",
-            "events_url": "https://api.github.com/users/pelya/events{/privacy}",
-            "received_events_url": "https://api.github.com/users/pelya/received_events",
-            "type": "User",
-            "name": "Sergii Pylypenko",
-            "description": "Port of SDL library and several games to the Android OS.",
-            "location": "Kiev",
-            "site_admin": false,
-            "score": 1.0
-        },
-        {
-            "login": "Metnew",
-            "id": 10862289,
-            "node_id": "MDQ6VXNlcjEwODYyMjg5",
-            "avatar_url": "https://avatars1.githubusercontent.com/u/10862289?v=4",
-            "gravatar_id": "",
-            "url": "https://api.github.com/users/Metnew",
-            "html_url": "https://github.com/Metnew",
-            "followers_url": "https://api.github.com/users/Metnew/followers",
-            "following_url": "https://api.github.com/users/Metnew/following{/other_user}",
-            "gists_url": "https://api.github.com/users/Metnew/gists{/gist_id}",
-            "starred_url": "https://api.github.com/users/Metnew/starred{/owner}{/repo}",
-            "subscriptions_url": "https://api.github.com/users/Metnew/subscriptions",
-            "organizations_url": "https://api.github.com/users/Metnew/orgs",
-            "repos_url": "https://api.github.com/users/Metnew/repos",
-            "events_url": "https://api.github.com/users/Metnew/events{/privacy}",
-            "received_events_url": "https://api.github.com/users/Metnew/received_events",
-            "type": "User",
-            "name": "Vladimir Metnew",
-            "description": 'null',
-            "location": "Ukraine, Kiev",
-            "site_admin": false,
-            "score": 1.0
-        }
-    ];
+    constructor(props) {
+        super(props);
+        this.state = {
+            users: []
+
+
+        };
+    }
+     componentDidMount() {
+        fetch('https://api.github.com/search/users?q=location:kiev&sort=followers&order=desc&per_page=10&page=1')
+            .then(response => response.json())
+            .then(users => this.setState(users));
```

Comment:
You should also catch `error` and `console.error(error)` it

Reply:
Also, state is an object, but you are putting your users response there instead of object.
It should look like this:
`this.setState({someField: someValue})`